api/src/main/java/org/pmiops/workbench/api/ProfileController.java

```diff
@@ -281,19 +281,10 @@ private User initializeUserIfNeeded() {
   }
 
   @Override
-  public ResponseEntity<Void> deleteAccount() {
-    UserAuthentication userAuth =
-        (UserAuthentication)SecurityContextHolder.getContext().getAuthentication();
-    String email = userAuth.getPrincipal().getEmail();
-    String[] parts = email.split("@");
-    try {
-      directoryService.deleteUser(parts[0]);
-    } catch (IOException e) {
-      throw ExceptionUtils.convertGoogleIOException(e);
-    }
-
-    userDao.delete(userDao.findUserByEmail(email));
-
+  public ResponseEntity<Void> disableAccount() {
+    User user = userProvider.get();
+    user.setDisabled(true);
```

Comment:
Do we need to have a way to re-enable your account once it's been disabled? (I'm assuming that's out of scope for this PR in any case...)

Do we need to do anything to your Firecloud or Google account when your account becomes disabled? For example, should we take you out of whatever authorization domains you're a part of?

Reply:
Yes, I was planning on doing re-enabling as a follow up. 

I will add revoking your access to auth domains as a part of this PR. Good catch.